### ISSUE TITLE
Quote Submission

### ISSUE THREAD

**Issue by kicohen** [ABTech/abtt#353]
Can we make it so that quotes cannot be Printed, Previewed or Emailed until they have been approved by a Exec (Finance) user or a Head of Tech user.

This would be a button on an invoice that when clicked, sends an email to all Exec (Finance) and the Head of Tech saying an invoice is waiting for approval. The invoice would than have a button saying that Approved or Revise. Both cases would email the TiC saying that the invoice has either been approved or that it needs to be revised. If revised a not should be provided. Approved quotes can then be Printed, Previewed or Emailed.

**Comment by agotsis** [comment]
This sounds incredibly annoying and distrustful of our own members...

**Comment by plima101** [comment]
This is a business process. This is how any real business operates it finances (https://en.wikipedia.org/wiki/Maker-checker). It's not distrustful, its an extra layer of confirmation and as our organization grows its important to have checks on everything we do.

**Comment by EmNewman** [comment]
Seems like a lot of extra work for the 3(?) people that have to do that. Unlike the "checker" for a real business, Tech isn't our full time job. Just don't make it a feature that's impossible to revert, cause I can totally see this getting overwhelming/slowing things down when we get busy during the semester. Maybe even toggle-able.

**Comment by adamonster97** [comment]
This is a final check to ensure consistency in billing. I feel that this is a minor change that can go a long way in standardizing prices while still allowing TiCs the ability to make invoices for their events.

If it's important to have consistent pricing and billing, this is the best way to accomplish that.

**Comment by timparenti** [comment]
I agree that, since consistent pricing is a clear and stated goal, this is one of the cleanest and most effective ways to achieve it without having to entrench too much of the process into Tracker itself (as #351 might instead require).  It would also be fairly straightforward to implement in a way that could be disabled at the option of later Executive Boards.

Should also be pointed out, though, that this nominally requires something like #352 to work.
